src/implementations/twig/components/description-list/description-list.story.js

```diff
@@ -9,66 +7,59 @@ import dataDescriptionListHorizontal from '@ecl/specs-component-description-list
 import descriptionList from './description-list.html.twig';
 import notes from './README.md';
 
-const prepareList = (data) => {
-  if (data.variant) {
-    data.variant = select(
-      'variant',
-      [data.variant],
-      data.variant,
-      tabLabels.required
-    );
-  }
-  data.items.forEach((item, i) => {
-    if (Array.isArray(item.term)) {
-      item.term.forEach((termItem, j) => {
-        data.items[i].term[j] = text(
-          `items[${i}].term[${j}]`,
-          termItem,
-          tabLabels.required
-        );
-      });
-    } else {
-      item.term = text(`items[${i}].term`, item.term, tabLabels.required);
-    }
-    if (Array.isArray(item.definition)) {
-      item.definition.forEach((definitionItem, k) => {
-        data.items[i].definition[k] = text(
-          `items[${i}].definition[${k}]`,
-          definitionItem,
-          tabLabels.required
-        );
-      });
-    } else {
-      item.definition = text(
-        `items[${i}].definition`,
-        item.definition,
-        tabLabels.required
-      );
-    }
-  });
-
-  getExtraKnobs(data);
+const getArgTypes = (data) => {
+  return {
+    term: {
+      name: 'first item term',
```

Comment:
Think it would be better to label the control as the property, in general, and if possible. Here it could be:
`term (first item)`
`definition (first item)`